libs/core/src/lib/combobox/combobox.component.html

```diff
@@ -72,7 +73,7 @@
                     fd-list-item
                     tabindex="0"
                     class="fd-combobox-list-item"
-                    [selected]="displayFn(term) === inputText"
+                    [selected]="displayFn(term) === searchInputElement.value"
```

Comment:
Regarding to our discussion about performance, we should avoid using methods in templates,
in this case, I think BehaviourSubject would be better

Reply:
Maybe not be the best place for that, because the idea behind having displayFn be an Input() here is to allow application developers to provide their own functions that handle displaying the string.  If we used a behavior subject here, rather than just accepting a function that returns a string, we would be adding another layer of complexity we're going to have to explain ourselves about